client/src/api/socketService.js

```diff
@@ -0,0 +1,18 @@
+import openSocket from 'socket.io-client';
+
+const socket = openSocket('http://localhost:9000/');
```

Comment:
Why strictly connecting to the `localhost`? This should be replaced by an environment variable which will be a server URL in case of production and `localhost` in case of development.